src/qri/versionInfo.ts

```diff
@@ -36,6 +36,12 @@ export interface VersionInfo {
   runID?: string
   runStatus?: RunStatus
   runDuration?: number
+
+  runCount?: number // RunCount is the number of times this dataset's transform has been run
+  commitCount?: number // CommitCount is the number of commits in this dataset's history
+  downloadCount?: number // DownloadCount is the number of times this dataset has been directly downloaded from this Qri node
+  followerCount?: number // FollowerCount is the number of followers this dataset has on this Qri node
+  openIssueCount?: number // OpenIssueCount is the number of open issues this dataset has on this Qri node
```

Comment:
I'd much rather follow the golang convention of using a default value for the number instead of making these fields optional. To accomplish that in typescript make all of these fields required (remove the `?`), ensure all versionInfo instantiation passes through the `newVersionInfo` constructor, and add `|| 0` to the end of each value assignment. Eg: `runCount: data.runCount || 0`.

This removes the need for existential checks all of the place. It won't be true for all verionInfo fields yet, but hey, gotta start somewhere.